packages/fast-components-styles-msft/src/utilities/high-contrast.ts

```diff
@@ -110,6 +130,14 @@ export const highContrastForeground: CSSRules<DesignSystem> = {
     },
 };
 
+// Used to set foreground and glyph to be 'select-text' color
+export const highContrastSelectionForeground: CSSRules<DesignSystem> = {
+    [highContrastSelector]: {
+        color: "HighlightText !important",
```

Comment:
Can all values in this file use the constants? Maybe add `important` to each color anyway.

Reply:
create a function called `ImportantColor` to use here